dblookupext/esdtSupply/esdtSuppliesProcessor.go

```diff
@@ -56,6 +56,10 @@ func (sp *suppliesProcessor) ProcessLogs(blockNonce uint64, logs map[string]data
 
 // RevertChanges will revert supplies changes based on the provided block body
 func (sp *suppliesProcessor) RevertChanges(header data.HeaderHandler, body data.BodyHandler) error {
+	if check.IfNil(header) || check.IfNil(body) {
```

Comment:
Interesting. There might be cases when the provided header is not nil but the body is (in case of empty blocks) but then, we don't have to actually do a revert because there were no transactions. I guess this holds for all cases we care about.